src/halfpipe/tui/templates/feature_template.py

```diff
@@ -0,0 +1,643 @@
+            self.get_widget_by_id("bandpass_filter_hp_width").styles.visibility = "hidden"
+            # The visibility of the input box of SwitchWithInputBox widget is on mount set by presence of string,
+            # since a string can be passed as a default value, we need to also override the input_switch_input_box subwidget
+            # of the SwitchWithInputBox widget
+            self.get_widget_by_id("bandpass_filter_lp_width").get_widget_by_id(
+                "input_switch_input_box"
+            ).styles.visibility = "hidden"
+            self.get_widget_by_id("bandpass_filter_hp_width").get_widget_by_id(
+                "input_switch_input_box"
+            ).styles.visibility = "hidden"
+            self.get_widget_by_id("preprocessing").styles.height = 26
+            self.get_widget_by_id("confounds_selection").styles.offset = (0, -5)
+            self.temp_bandpass_filter_selection = copy.deepcopy(self.setting_dict["bandpass_filter"])
+            self.setting_dict["bandpass_filter"]["type"] = None
+
+    @on(SwitchWithSelect.Changed, "#bandpass_filter_type")
+    def _on_bandpass_filter_type_changed(self, message) -> None:
+        """
+        Handles changes in the bandpass filter type.
+
+        This method is called when the value of the `SwitchWithSelect`
+        widget with the ID "bandpass_filter_type" changes. It updates the
+        bandpass filter settings in `setting_dict` based on the selected
+        filter type (Gaussian or frequency-based).
+
+        Parameters
+        ----------
+        message : SwitchWithSelect.Changed
+            The message object containing information about the change.
+        """
+        bandpass_filter_type = message.value
+        if message.control.switch_value is True:
+            self.set_bandpass_filter_values_after_toggle(bandpass_filter_type)
+
+    def set_bandpass_filter_values_after_toggle(self, bandpass_filter_type) -> None:
+        """
+        Sets the bandpass filter values after toggling the filter type.
+
+        This method is called after the bandpass filter type is toggled.
+        It updates the labels and values of the low-pass and high-pass
+        filter widgets based on the selected filter type. It also updates
+        the `setting_dict` with the new filter values.
+
+        Parameters
+        ----------
+        bandpass_filter_type : str
+            The selected bandpass filter type ("gaussian" or
+            "frequency_based").
+        """
+        if bandpass_filter_type == "frequency_based":
+            lowest_value = (
+                self.setting_dict["bandpass_filter"]["low"]
+                if "low" in self.setting_dict["bandpass_filter"] and self.setting_dict["bandpass_filter"]["low"] is None
+                else str(self.setting_dict["bandpass_filter"]["low"])
+                if "low" in self.setting_dict["bandpass_filter"]
+                else "0.01"
```

Comment:
Could you also add these four default values to the standards?

Reply:
To clarify, there are four default values in this file, all in the last `else` branch

Reply:
yes, two defaults for gaussian and two for frequency, this is because when user toggles with the filter type selection I need to give there some default values, I now piped the values from standards

Reply:
Thanks!